lib/infra/infra-stack.ts

```diff
@@ -475,30 +482,26 @@ export class InfraStack extends Stack {
           cwd: '/home/ec2-user',
           ignoreErrors: false,
         }));
+        cfnInitConfig.push(InitCommand.shellCommand('set -ex;cd opensearch;sudo -u ec2-user bin/opensearch-plugin install repository-s3 --batch', {
```

Comment:
Hi @rishabh6788 ,

Instead of manually adding the plugin name here, can we take a list of native plugins to be installed and then install in one go?
Right now its repository-s3 and discovery-ec2. In future there might be more. 

Something like:
native-plugins-to-install: repository-s3, discovery-ec2?